EndlessSky.xcodeproj/project.pbxproj

```diff
@@ -27,7 +28,6 @@
 		62A405BA1D47DA4D0054F6A0 /* FogShader.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 62A405B81D47DA4D0054F6A0 /* FogShader.cpp */; };
 		62C3111A1CE172D000409D91 /* Flotsam.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 62C311181CE172D000409D91 /* Flotsam.cpp */; };
 		6A5716331E25BE6F00585EB2 /* CollisionSet.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 6A5716311E25BE6F00585EB2 /* CollisionSet.cpp */; };
-		938E07E4235970CE00DC2C2B /* FormationPattern.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 938E07E0235970CE00DC2C2B /* FormationPattern.cpp */; };
```

Comment:
is this normal? Was it added twice? I mean the test runs I guess but feels weird to me

Reply:
i dont really know, xcode will be gone soon anyways

Reply:
The patterning-code is the only part of formation flying that was merged so far, so I guess this file is just not referenced from other files yet (in which case the game can compile fine without this file).

Reply:
xcode is removed now anyways